Automattic/jetpack
# Social: Fix the (no title) display in the post list screen.

Fix the (no title) display in the post list screen. I have branched this on top of https://github.com/Automattic/jetpack/pull/35462

## Proposed changes:

* Use the `the_title` hook, so, (no title) doesn't show up in the post list screen. 

### Other information:


## Jetpack product discussion


p1707313242731969/1707307223.503819-slack-C02JJ910CNL
## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






* On trunk, create a new social note. Go to the Social Notes list view screen and it will look like below. 
<img width="889" alt="Screenshot 2024-02-07 at 8 48 35 PM" src="https://github.com/Automattic/jetpack/assets/6594561/3141d887-4cb5-4024-8b2b-04f366061edb">

* On this branch, we have fixed that. Go to the Social Notes list view to ensure that. 
<img width="886" alt="Screenshot 2024-02-07 at 8 49 04 PM" src="https://github.com/Automattic/jetpack/assets/6594561/99bdf4d3-20e4-4337-8714-65264bdf4c8a">
